.github/workflows/python-package-windows.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   build:
 
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
Please add a comment why this change is necessary.

Reply:
@matteocao did this to fix the tests not passing on Windows with torch 1.13.

Reply:
I made tests with a VM from Azure, win11 with all packages. All tests pass.

However on the CI VM, which is a version of. windows of 2022, it does not unexplicably. 

hence, I changed the VM: by the way, it goes also so much faster! hence, I believe the CI image may have some issues. In conclusion, I think it is smarted to change the CI image where tests pass properly (as they should) and wait for a new image to change this label back to `latest`.